src/main/groovy/com/google/protobuf/gradle/GenerateProtoTask.groovy

```diff
@@ -214,38 +236,58 @@ public class GenerateProtoTask extends DefaultTask {
     this.fileResolver = fileResolver
   }
 
+  @Internal
   SourceSet getSourceSet() {
     Preconditions.checkState(!Utils.isAndroidProject(project),
         'sourceSet should not be used in an Android project')
     Preconditions.checkNotNull(sourceSet, 'sourceSet is not set')
     return sourceSet
   }
 
+  @Nullable
+  @Optional
+  @Input
+  String getSourceSetName() {
```

Comment:
Why is the name of the source set an input?

Reply:
Right, that doesn't make sense. I just removed it.